apps/financial-aid/web-osk/src/components/ServiceCenter/ServiceCenter.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Text } from '@island.is/island-ui/core'
+
+import {
+  ContentContainer,
+  Footer,
+} from '@island.is/financial-aid-web/osk/src/components'
+
+import { ServiceCenter as IServiceCenter } from '@island.is/financial-aid/shared/data'
+import { useLogOut } from '@island.is/financial-aid-web/osk/src/utils/useLogOut'
+
+interface Props {
+  serviceCenter?: IServiceCenter
+}
+
+const ServiceCenter = ({ serviceCenter }: Props) => {
+  const logOut = useLogOut()
+
+  return (
+    <>
+      <ContentContainer>
+        <Text as="h1" variant="h2" marginBottom={[1, 1, 2]}>
+          Fjárhagsaðstoð Hafnarfjarðar
+        </Text>
+
+        <Text marginBottom={[1, 1, 2]}>
+          Athugaðu að á þessum tímapunkti er þessi umsókn um fjárhagsaðstoð
+          eingöngu ætluð íbúum Hafnarfjarðar.
+        </Text>
+
+        {serviceCenter && (
+          <Text>
+            Athugaðu að {serviceCenter?.name} er ekki orðið hluti af lausninni
```

Comment:
Should we send just the name of the service center instead of the whole object to the component since it is the only thing we are using?

Reply:
I just added name for now, but I think in the future we will be using more information from service, homePage for example when that information has been collected 🤓  Should I chnage we take name in for now?

Reply:
Your call, I usually don't plan for the future when coding, cause those plans change